src/Verkehr/Network.kt

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by Sven on 09.08.2017.
+ */
+
+class Network {
+
+    /**
+     * initialize of the variables. start(?) value is zero
+     * variable cars represent the list/ number of cars
+     * constructor(from java).. try to explain in english: for initialisation of values which are relevant for our object (here. cars and capacity)
```

Comment:
Kenne von JavaScript die constructo r Methode. Ist gerade in Arbeit. Kommt morgen. Hatte es schon kommentiert, nicht verwirren lassen ;)